Sources/StreamChat/Repositories/AuthenticationRepository.swift

```diff
@@ -200,7 +200,7 @@ class AuthenticationRepository {
             return
         }
 
-        scheduleTokenFetch(isRetry: false, userInfo: nil, tokenProvider: tokenProvider, completion: completion)
+        scheduleTokenFetch(isRetry: true, userInfo: nil, tokenProvider: tokenProvider, completion: completion)
```

Comment:
Would be nice to provide more details on the PR, to understand how you came up with the fix 🙏